RELEASE_NOTES.md

```diff
@@ -1,3 +1,39 @@
+## Cookstyle 6.12
+
+### 3 New Cops
+
+#### ChefDeprecations/ChefDKGenerators
+
+The `ChefDeprecations/ChefDKGenerators` cop detects custom cookbook generators for the `chef` CLI that utilize the legacy `ChefDK` classes and not `ChefCLI` classes.
```

Comment:
```suggestion
The `ChefDeprecations/ChefDKGenerators` cop detects custom cookbook generators for the `chef` CLI that utilize the legacy `ChefDK` classes and instead of the `ChefCLI` classes.
```